pkg/log/log.go

```diff
@@ -402,7 +393,15 @@ func (l logger) Panic(format string, args ...interface{}) {
 }
 
 func (l logger) DebugBlock(prefix string, format string, args ...interface{}) {
-	l.block(LevelDebug, prefix, format, args...)
+	log.RLock()
+	suppress := !log.forced
+	if !suppress {
+		_, suppress = log.debug[l]
+	}
```

Comment:
What is the logic here 🤔  If global forced debug logging is turned on we take `suppress` from the individual debug setting??

Reply:
I just plain broke the logic in a last minute change that was supposed to be too small to go wrong... Thanks for spotting.